web/templates/settings/edit_bot_form.hbs

```diff
@@ -56,7 +56,7 @@
         </button>
```

Comment:
`deactivate_bot_button` just above looks like it should be updated with the rest of these?